packages/core/src/index.ts

```diff
@@ -1,11 +1,12 @@
 /**
  * @module @aerogearservices/core
+ *
+ * Module used to aggregate common functionalities for AeroGear Services SDK's
  */
 import console from "loglevel";
 console.setDefaultLevel(console.levels.WARN);
 
 // Configuration parsers
-export * from "./configuration";
+export * from "./config";
 
-// Native platform metrics
-export *  from "./metrics";
+export * from "./Core";
```

Comment:
I think we shouldn't export `AgsCore`, only `coreInstance`.

Reply:
Moving to singleton. Still required